app/views/templates/search/search_object.rb

```diff
@@ -629,40 +629,107 @@ def media_type(url)
         end
       end
 
+      def download_disabled(rights)
+        disabled = false
+        [ 'http://www.europeana.eu/rights/rr-p',
+          'http://www.europeana.eu/rights/rr-r/' ].map do | blacklisted |
+            if rights.index(blacklisted) == 0
+              disabled = true
+            end
+        end
+        disabled
+      end
+
       def simple_rights_label_data(rights)
         return nil unless rights.present?
         # global.facet.reusability.permission      Only with permission
         # global.facet.reusability.open            Yes with attribution
         # global.facet.reusability.restricted      Yes with restrictions
 
-        prefix = t('global.facet.header.reusability') + ' '
-
         if rights.nil?
           nil
+        elsif rights.index('http://creativecommons.org/publicdomain/zero') == 0
+         {
+           license_human: t('global.facet.reusability.open'),
+           license_name: t('global.facet.reusability.advanced-cc0'),
+           license_CC0: true
+        }
+        elsif rights.index('http://creativecommons.org/licenses/by/') == 0
+         {
+           license_human: t('global.facet.reusability.open'),
+           license_name: t('global.facet.reusability.advanced-cc-by'),
+           license_CC_BY: true
+        }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.